root/iswc/Index.js

```diff
@@ -0,0 +1,130 @@
+
+type PropsT = {|
+  +$c: CatalystContextT,
+  +iswcs: $ReadOnlyArray<IswcT>,
+  +works: $ReadOnlyArray<WorkT>,
+|};
+
+const Index = ({$c, iswcs, works}: PropsT) => {
+  const userExists = $c.user_exists;
+  const iswc = iswcs[0];
+  return (
+    <Layout fullWidth title={l('ISWC “{iswc}”', {iswc: iswc.iswc})}>
+      <h1>
+        {l('ISWC “{iswc}”',
+          {__react: true, iswc: <CodeLink code={iswc} key="iswc" />})}
+      </h1>
+      <h2>
+        {ln(
+          'Associated with {num} work',
+          'Associated with {num} works',
+          works.length,
+          {num: works.length},
+        )}
+      </h2>
+      <form action="/work/merge_queue" method="post">
+        <table className="tbl">
+          <thead>
+            <tr>
+              {userExists ? (
+                <th style={{width: '1em'}}>
+                  <input type="checkbox" />
+                </th>
+              ) : null}
+              <th>{l('Title')}</th>
+              <th>{l('Writers')}</th>
+              <th>{l('Artists')}</th>
+              <th>{l('Type')}</th>
+              <th>{l('Language')}</th>
+            </tr>
+          </thead>
+          <tbody>
+            {works.map((work, index) => (
+              <tr
+                className={loopParity(index)}
+                key={work.id}
+              >
+                {userExists ? (
+                  <td>
+                    <input
+                      name="add-to-merge"
+                      type="checkbox"
+                      value={work.id}
+                    />
+                  </td>
+                ) : null}
+                <td><EntityLink entity={work} /></td>
+                <td>
+                  <ul>
+                    {work.writers.map(r =>(
+                      <li key={r.entity.id}>
+                        {l('{artist} ({roles})', {
+                          __react: true,
+                          artist: <EntityLink entity={r.entity} />,
+                          roles: r.roles.length > 1
+                            // $FlowFixMe
+                            ? commaOnlyList(uniq(r.roles))
+                            : r.roles[0],
+                        })}
+                      </li>
+                    ))}
+                  </ul>
+                </td>
+                <td>
+                  <ul>
+                    {work.artists.map((r, i) => (
+                      <li key={i}>
+                        <ArtistCreditLink artistCredit={r} />
+                      </li>
+                    ))}
+                  </ul>
+                </td>
+                <td>{work.typeName ? lp_attributes(work.typeName, 'work_type') : null}</td>
+                <td>
+                  <ul>
+                    {work.languages.map(r => (
+                      <li key={r.language_id}>
+                        <abbr title={l_languages(r.language.name)}>
+                          {r.language.iso_code_3}
+                        </abbr>
```

Comment:
There is enough place in this page to display language name rather than just its code.

Reply:
Small fixes/improvements are welcome too as long as they are not doubtful.
As you stated it is currently broken, parity is not even wished here. :wink:

Reply:
I agree with that bit, but just thought it might make sense to have consistency in all search results (so, have all language/script shown normally or with the code). At the very least, I'd want this one to match the Work search results (where we currently also show code only). Would you (and @mwiencek) suggest to change both to use work name instead?

Reply:
I'm okay with changing both if it's not difficult.

Reply:
It's trivial (after all, we already load the name for the abbr hover). Should we still display the codes in any way, if we change this?

Ideally we would also change the release results, but language and script *do* compete with a lot of other things in that table. I'm just thinking that if that's the only place we use the abbreviations, it might feel kinda strange.

Reply:
Yeah, not sure about the release results, and it would be strange if they were inconsistent. Let's just keep parity in this PR?

Reply:
Honestly I just feel like the whole `abbr` is buggish, but it ain't the main purpose of this PR.